src/rate/logging/logger.decorator.ts

```diff
@@ -0,0 +1,27 @@
+import { Logger } from '@nestjs/common';
+
+export function Log(targetUrl: string) {
```

Comment:
Maybe it would be more convenient to add something related to rate to the name ? For example: LogRateDecorator, LogRateRequest, etc

Reply:
Good catch!